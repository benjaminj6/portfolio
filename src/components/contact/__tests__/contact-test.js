/* eslint-env jest */
import { h } from 'preact'
import { shallow } from 'preact-render-spy'

import Contact from '../'

test('Contact snapshot', () => {
  const tree = shallow(<Contact />)
  expect(tree).toMatchSnapshot()
})
