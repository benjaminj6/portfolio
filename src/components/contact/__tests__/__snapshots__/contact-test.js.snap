// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<div class="Contact">
  <p>
    Let's chat! I don't mind if you're a recruiter, senior dev, junior dev, or just getting started, I'm always open for a conversation. 
    <A href="mailto:benjamin.d.johnson@icloud.com">Shoot me an email</A>
     or connect with me on 
    <A
      href="https://www.linkedin.com/in/benjamin-d-johnson/"
      target="_blank"
    >
      LinkedIn
    </A>
     and let's get the conversation started.
  </p>
</div>

`;
